### PR TITLE
feat(adir): correct default value for speeds discrete signals

### PR DESCRIPTION
**Pull request by crocket63**
Fixes #[issue_no]

Sets expected discrete signal values for speed warnings of ADIRs when they are fault or off.

When ADIR 1 and 3 are off, gear safety valve will close and gear become inoperative.

## Additional context

Discord username (if different from GitHub):

## Testing instructions

- When ADIR 1 and 3 are off, gear shall be inoperative.

- In nominal situation gear works and stop working above 260kts

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: src/systems/systems/src/navigation/adirs.rs [hunk 1]
```diff
@@ -535,29 +535,32 @@ impl AirDataInertialReferenceUnit {
         self.ir.has_fault()
     }
 
+    // If above speed threshold OR if data is unavailable: all discrete are set to TRUE
     fn update_discrete_outputs(&mut self) {
         let speed_knot = self.adr.computed_airspeed_raw().get::<knot>();
 
+        let data_unavailable = self.ir_has_fault() || !self.ir_is_on() || !self.is_fully_aligned();
```
[the remark this answers is not in the document]
Comment (crocket63) [reply src/systems/systems/src/navigation/adirs.rs]: Switched to adr side only however nothing is implemented yet for ir or adr

**Comment by johnpmaguire** [comment]
Quality Assurance Report
Discord john P Maguire#9027
Object of testing: #6808
Tier of Testing: 1
Date: 22/02/2022

Testing Process:

In experimental build O7ff876. Noted the gear moved on loss of ADIR 1 and 3.
Installed PR #7458 and spawned on the runway at EGLL. Completed a takeoff and leveled off at 220kts. Cycled the gear several times in the following states. 

ADIR state       Result
1,2,3 On          Gear cycled normally
1,3 off 2 on     Gear in op
1 off 2, 3 on    Gear cycled normally
3 off 1,2 on     Gear cycled normally

Moving the gear lever and leaving out of position then aligning one of the ADIRs. All combinations of 1 and 3 were tried.
1 and 3 off. Then 1 or 3 back on. The gear lever moved up while the gear down.  Gear remained down.
                                                      The gear lever moved down while gear was up. Gear lowered on alignment of 1 or 3.

Negatives
None noted.

Testing Results
Passed

Conclusions 
The operation of the gear safety valve works as described. No testing instructions were included for after realignment. If the documentation is as observed for the misalignment of lever and gear, then all good

**Comment by johnpmaguire** [comment]
Quality Assurance Report
Discord john P Maguire#9027
Object of testing: #6808
Tier of Testing: 1
Date: 22/02/2022

Testing Process:

Flew the route in experimental build O7ff876. Noted the gear moved on loss of ADIR 1 and 3.
Installed PR #7458 and spawned on the runway at EGLL. Completed a takeoff a leveled off at 220kts. Cycled the gear several times in the following states. 

ADIR state       Result
1,2,3 On          Gear cycled normally
1,3 off 2 on     Gear in op
1 off 2, 3 on    Gear cycled normally
3 off 1,2 on     Gear cycled normally

Moving the gear lever and leaving out of position. All combinations of 1 and 3 were tried.
1 and 3 off. Then 1 or 3 back on. The gear lever moved up while the gear down.  Gear remained down.
                                                      The gear lever moved down while gear was up. Gear lowered on alignment of 1 or 3.

Negatives
None noted.

Testing Results
Passed

Conclusions 
The operation of the gear safety valve works as described. No testing instructions were included for after realignment. If the documentation is as observed for the misalignment of lever and gear, then all good

**Review by lukecologne — APPROVED** [review]
Looks good now!

**Pull request merged**